src/backend/reach_backend/Api/serializers.py

```diff
@@ -34,6 +35,35 @@ class Meta:
         fields = "__all__"
 
 
+class UserDataRegistrationSerializer(serializers.ModelSerializer):
+    """Serializer for the user data registration model."""
+
+    class Meta:
+        """Meta class for user data registration serializer."""
+
+        model = UserData
+        fields = ("first_name", "last_name", "is_clinician")
```

Comment:
Do we need this? What's its purpose? The way I thought it would go is after registering an account, we just make 2 post requests - one for the actual registration, and one to create a "UserData" instance, which will point to a user (i..e, the one that was just created).

Reply:
That serializer is essentially just for combining the creation of userData and user. If we use 2 post requests then we would also have to add safeguards to ensure that every user has userData associated with it while with one we can assume that this is true. We are not really handling super complex data so I think 1 post should be fine but I'm not really sure.

Reply:
Alright let's go with what u got then.